test/change.test.js

```diff
@@ -82,6 +82,36 @@ describe('Change', function() {
     });
   });
 
+  describe('Change.rectifyModelChanges - promise variant', function() {
+    describe('using an existing untracked model', function() {
+      beforeEach(function(done) {
+        var test = this;
+        Change.rectifyModelChanges(this.modelName, [this.modelId])
+          .then(function(trackedChanges) {
+            done();
+          })
+          .catch(function(err) {
+            return done(err);
+          });
```

Comment:
Mocha supports promises OOTB, no need to install dummy handlers. Notice that the function passed to `beforeEach` does not take any arguments.

```js
beforeEach(function() {
  return Change.rectifyModelChanges(this.modelName, [this.modelId]);
});
```

BTW for future reference, `then` function takes an optional second argument `errorHandler`, so if we wanted to install both success & error handler, this is a better way:

```js
Change.rectifyModelChanges(...).then(
  function onSuccess() { done(); },
  function onError(err) { done(err); });

// which can be further simplified to

Change.rectifyModelChanges(...).then(
  function onSuccess(trackedChanges) { done(); },
  done);
```